mtmount/Dockerfile

```diff
@@ -0,0 +1,111 @@
+# This docker file cannot fully automate the EUI installation. You must carry on
+# with the installation of the VIPM package installs to complete the installation. 
+
+FROM centos:centos7
+
+LABEL maintainer Andrew Heyer <aheyer@lsst.org>
+
+ENV USER ${USER:-saluser}
+ENV HOME /home/saluser
+USER root
+# Create user and group
+RUN groupadd lsst && adduser -u 1004 -m -g lsst -s /bin/bash saluser
+
+COPY lsst-ts-nexus.repo /etc/yum.repos.d/lsst-ts-nexus.repo
+COPY wandisco-git.repo /etc/yum.repos.d/wandisco-git.repo
+RUN rpm --import http://opensource.wandisco.com/RPM-GPG-KEY-WANdisco
+RUN yum install -y git wget
+RUN yum install -y OpenSpliceDDS-6.9.0
+
+#Install and update various packages needed for Labview, VPM, and be able to
+#compile C++ tma management 
+#RUN yum update
+RUN yum groupinstall -y "Development Tools"
+RUN yum install -y cmake boost-devel.x86_64
+RUN wget http://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
```

Comment:
My preferred method is:
```bash
yum install -y epel-release
```
looks more pro than the two lines (also note use of wget, which is in Centos and also Ubuntu fading away)

Reply:
When making this change I get the following error 
```Step 14/57 : RUN yum install -y epel-release
 ---> Running in 86ef1377fdb6
Loaded plugins: fastestmirror, ovl
Determining fastest mirrors
 * base: mirror.orbyta.com
 * extras: mirror.orbyta.com
 * updates: mirror.orbyta.com
Resolving Dependencies
--> Running transaction check
---> Package epel-release.noarch 0:7-11 will be installed
--> Finished Dependency Resolution

Dependencies Resolved

================================================================================
 Package                Arch             Version         Repository        Size
================================================================================
Installing:
 epel-release           noarch           7-11            extras            15 k

Transaction Summary
================================================================================
Install  1 Package

Total download size: 15 k
Installed size: 24 k
Downloading packages:


Error downloading packages:
  epel-release-7-11.noarch: Insufficient space in download directory /var/cache/yum/x86_64/7/extras/packages
    * free   0
    * needed 15 k

The command '/bin/sh -c yum install -y epel-release' returned a non-zero code: 1```